docs/introduction.md

```diff
@@ -0,0 +1,50 @@
+# Introduction
```

Comment:
Can this bit go on the repo README (or the /docs README)? IMO the default README.md = introduction